### DISCUSSION TITLE
lean-af-fuel-economy/

### DISCUSSION THREAD

**Discussion by giscus** [n0vad3v/blog-comments#5] [Ideas]
# lean-af-fuel-economy/

在一个名为「AFR 16」的小圈子中偶然看到「冬夏」提到了一个更稀的空燃比降低油耗的思路，打算为此进行验证。 申明：本文是对于 AF 以及一些相关的知

https://nova.moe/lean-af-fuel-economy/

**Comment by tian24880** [comment]
谢谢分享

**Comment by HCXI2000** [comment]
感谢分享
